README.md

```diff
@@ -234,6 +234,10 @@ The following is a list of static resources.
 - [azure_security_center_policies](docs/resources/azure_security_center_policies.md)
 - [azure_sentinel_incidents_resource](docs/resources/azure_sentinel_incidents_resource.md)
 - [azure_sentinel_incidents_resources](docs/resources/azure_sentinel_incidents_resources.md)
+- [azure_service_bus_namespace](docs/resources/azure_service_bus_namespace.md)
+- [azure_service_bus_namespaces](docs/resources/azure_service_bus_namespaces.md)
```

Comment:
These are not required. these are already there in the https://github.com/inspec/inspec-azure/pull/562/files PR.